syntax = "proto3";

package Car_msg;

message ruicheng_control{
    int32 TargetSteeringAngle = 1;
    int32 TargetSpeed = 2;
    int32 TargetLedStatus = 3;
    int32 LightSwitch= 4;
    int32 SpeedGear = 5;
}

message ruicheng_feedback{
    int32 EmBrakerStatus = 1;
    int32 EpsStatus = 2;
    int32 TorqueStatus = 3;
    int32 DecStatus = 4;
    int32 SysStatus = 5;
    int32 ucGearCtrlStatus = 6;
    int32 BrakePedalStatus = 7;
    int32 CruiseStatus = 8;
    int32 ucGearPosition = 9;
    int32 uiSpeed = 10;
    float uiSteerAngle = 11;
}
